sdk/aqueduct/aqueduct_client.py

```diff
@@ -334,10 +334,11 @@ def publish_flow(
             retention_policy=retention_policy,
         )
 
-        if self._in_notebook_or_console_context:
-            _show_dag(self._api_client, dag)
-
         flow_id = self._api_client.register_workflow(dag).id
+
+        url = "http://" + self._api_client.aqueduct_address + "/workflow/" + str(flow_id)
+        print("UI: ", url)
```

Comment:
We should use the logger library here like the rest of the SDK.

Reply:
This Print statement was used to print for sdk output 
<img width="1042" alt="image" src="https://user-images.githubusercontent.com/61630725/178052960-1d3f65c4-c581-4bac-8af1-fe6b8991a342.png">
I tried to use logger but it would not print to the jupyter notebook.